lib/oneview-sdk/image-streamer/resource/api300/os_volumes.rb

```diff
@@ -17,6 +17,32 @@ module API300
       # OS Volumes resource implementation for Image Streamer
       class OsVolumes < Resource
```

Comment:
2 comments:
1.  Should this be `OSVolumes` instead of `OsVolumes`. The "S" is usually capitalized in OS. I understand that camelCase makes it a little weird, but we do it in other resources (e.g., 'SANManager')
2. CHANGELOG update?